app/views/alchemy/admin/pages/fold.js.erb

```diff
@@ -1,3 +1 @@
-$('#page_<%= @page.id -%>').replaceWith('<%= j render("page", page: @page) %>');
-Alchemy.watchForDialogs('#page_<%= @page.id %>');
-Alchemy.Spinner.watch('#page_<%= @page.id %>');
+Alchemy.Sitemap.fetch();
```

Comment:
I've been looking into this. It adds complexity to the code and no real performance benefit. Instead, I've optimized the code to not transfer folded branches of the tree to begin with.